doc/usermanual/preferences/darkroom_options.xml

```diff
@@ -101,11 +101,30 @@
       interpolation modes specified below, <quote>full (possibly slow)</quote> will use exactly
       the settings for full-size export (default <quote>at most ppg (reasonable)</quote>).
     </para>
-    
+
     <bridgehead renderas="sect5">reduce resolution of preview image</bridgehead>
 
     <para>
-      Decrease to speed up preview rendering, may hinder accurate masking.
+      This reduces the resolution of the preview window at the
+      upper-left of the screen. Consequently, calculations for
+      re-drawing this window are reduced to 1/4, 1/9 or 1/16 of the
+      main window. Sometimes coordinates for handles are borrowed from
```

Comment:
I would phrase it as: "Full scale may be needed for precise work on color pickers and/or masks."

Reply:
@pitbuster : this chunk has been written by a native English, so won't edit it :)

Reply:
At least the combination of "sometimes" and "rarely" in the same phrase in the same sentence is weird for a manual, but it is your call.